site-builder/src/main.rs

```diff
@@ -158,6 +171,11 @@ enum Commands {
     },
     /// Show the pages composing the site at the given object ID.
     Sitemap { object: ObjectID },
+    /// Preprocess the directory, creating and linking index files.
+    ///
+    /// This command allows to publish directories as sites. Warning: Rewrites all `index.html`
+    /// files.
+    Preprocess { path: PathBuf },
```

Comment:
Suggestion: I think we could be a bit more specific than "Preprocess", like "ListDirectory" or something like this. Also, the warning here and for the options above is only visible with "--help" but not with "-h".

Reply:
good points, changed.